hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -260,6 +260,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <version>3.3.0</version>
```

Comment:
version number should not be defined here, please do not add this change unless you have to.